schema/project-level/project-schema.json

```diff
@@ -1369,63 +1369,50 @@
     },
     "RelatedProject": {
       "title": "Related project",
-      "fdescription": "A reference to a project related to this project. Generally related projects either precede or follow on from the current project.",
+      "description": "A reference to a project related to this project. Generally related projects either precede or follow on from the current project.",
       "type": "object",
       "properties": {
         "id": {
           "title": "Relationship ID",
           "description": "A local identifier for this relationship, unique within this array.",
-          "type": "string"
+          "type": [
+            "string",
+            "integer"
```

Comment:
Remove 'integer' from `type`, per https://github.com/open-contracting/infrastructure/issues/184

(sorry - I know it was included in the example I suggested!)